src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/DefendantResponseApplicantNotificationHandler.java

```diff
@@ -179,7 +181,8 @@ PARTY_REFERENCES, buildPartiesReferences(caseData)
                 getPartyNameBasedOnType(caseData.getRespondent1())
                     .concat(" and ")
                     .concat(getPartyNameBasedOnType(caseData.getRespondent2())),
-                PARTY_REFERENCES, buildPartiesReferences(caseData)
+                PARTY_REFERENCES, buildPartiesReferences(caseData),
+                ALLOCATED_TRACK, toStringValueForEmail(caseData.getAllocatedTrack())
             );
         }
```

Comment:
need to include allocated track in test cases for DefendantResponseApplicantNotificationHandlerTest

Reply:
committed it now